sdk/assemblyscript/examples/anthropic-functions/assembly/index.ts

```diff
@@ -23,7 +23,7 @@ class StockPriceInput {
 // This model name should match the one defined in the modus.json manifest file.
 const modelName: string = "text-generator";
 
-export function getStockPrice(company: string, useTools: bool): string {
+export function getStockPrice(company: string, useTools: boolean): string {
```

Comment:
Should be just `bool` for AssemblyScript.

```suggestion
export function getStockPrice(company: string, useTools: bool): string {
```